blocks/email-composer/email-composer.js

```diff
@@ -16,6 +16,27 @@ export default async function decorate(block) {
     const token = await window.auth0Client.getTokenSilently();
 
     let project;
+
+    const googleProjectResponse = await fetch(`${SCRIPT_API}/list/${id}`, {
+      headers: {
+        authorization: `bearer ${token}`,
+      },
+    });
+    const googleProjectData = await googleProjectResponse.json();
+    project = googleProjectData?.project;
+
+    if (!project) {
+      // try dark alley
+      const darkAlleyProjectResponse = await fetch(`${SCRIPT_API}/darkAlleyList/${id}`);
+      const darkAlleyProjectData = await darkAlleyProjectResponse.json();
+      project = darkAlleyProjectData?.project;
+    }
+
+    // no project
+    if (!project) {
+      block.querySelector('.content p').textContent = OOPS;
```

Comment:
Add a return here or the code will continue to execute

Reply:
good catch👍